lib/Drush.php

```diff
@@ -0,0 +1,216 @@
+<?php
+
+/**
+ * @file
+ * Contains \Drush.
+ */
+
+use Drush\Symfony\BootstrapCompilerPass;
+
+use Symfony\Component\DependencyInjection\ContainerInterface;
+use Drupal\Core\DependencyInjection\ContainerNotInitializedException;
+
+/**
+ * Static Service Container wrapper.
+ *
+ * This code is analagous to the \Drupal class in Drupal 8.
+ *
+ * We would like to move Drush towards the model of using constructor
+ * injection rather than globals. This class serves as a unified global
+ * accessor to arbitrary services for use by legacy Drush code.
+ *
+ * Advice from Drupal 8's 'Drupal' class:
+ *
+ * This class exists only to support legacy code that cannot be dependency
+ * injected. If your code needs it, consider refactoring it to be object
+ * oriented, if possible. When this is not possible, and your code is more
+ * than a few non-reusable lines, it is recommended to instantiate an object
+ * implementing the actual logic.
+ *
```

Comment:
Based on your most recent comment, it sounds like Drush extensions are going to have to call Drupal::service() instead of getting Drupal dependencies injected. So this paragraph preaches an ideal which can't be met for Drush extensions. Let's remove or rephrase?